src/pretix/base/payment.py

```diff
@@ -442,6 +440,11 @@ def order_control_refund_perform(self, request: HttpRequest, order: Order) -> "b
                                     'back to the buyer manually.'))
 
 
+class PaymentException(Exception):
+    pass
+
+
+
```

Comment:
The style checker complains that this should be two, not three, blank lines :wink: